### ISSUE TITLE
[REC] Unholy DK Priority list not recommending Trinkets 

### ISSUE THREAD

**Issue by burnshadowz** [Hekili/hekili#2907]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

While following addon at no point is the addon recommending use of trinket. Checked action list at no point is the trinket action list called in the unholy priorities nor trinkets called during any other action list.

### How to Reproduce

1. Enter game
2. target Enemy/target dummy with on use trinket equiped
3. Follow advised rotation 
4. trinket never recommended.

### Snapshot (Link)

https://pastebin.com/38LtUXkZ

### Additional Information

```
build: v10.2.0-1.0.0i
level: 70 (70)
class: DEATHKNIGHT
spec: unholy

talents: BwPA/gf1fDREKLfAcG9Z2HkF/BAiQSicgISEkIRIIJJRCAAAAAAAAAKkEhIJpBAQiUEJJJSSA
    abomination_limb = 1/1
    antimagic_barrier = 1/1
    antimagic_shell = 1/1
    apocalypse = 1/1
    army_of_the_damned = 1/1
    army_of_the_dead = 1/1
    blinding_sleet = 1/1
    brittle = 1/1
    chains_of_ice = 1/1
    cleaving_strikes = 1/1
    dark_transformation = 1/1
    death_rot = 1/1
    death_strike = 1/1
    deaths_echo = 1/1
    defile = 1/1
    ebon_fever = 1/1
    empower_rune_weapon = 1/1
    enfeeble = 1/1
    epidemic = 1/1
    eternal_agony = 1/1
    festering_strike = 1/1
    festermight = 2/2
    gloom_ward = 1/1
    icebound_fortitude = 1/1
    icy_talons = 2/2
    improved_death_strike = 1/1
    improved_festering_strike = 2/2
    infected_claws = 1/1
    insidious_chill = 1/1
    magus_of_the_dead = 1/1
    merciless_strikes = 1/1
    might_of_thassarian = 1/1
    mind_freeze = 1/1
    morbidity = 2/2
    outbreak = 1/1
    permafrost = 1/1
    plaguebringer = 1/1
    raise_dead = 1/1
    raise_dead_2 = 1/1
    replenishing_wounds = 1/1
    rune_mastery = 2/2
    runic_attenuation = 1/1
    scourge_strike = 1/1
    sudden_doom = 1/1
    superstrain = 1/1
    unholy_assault = 1/1
    unholy_blight = 1/1
    unholy_bond = 1/2
    unholy_command = 2/2
    unholy_endurance = 1/1
    unholy_ground = 1/1
    unholy_pact = 1/1
    veteran_of_the_third_war = 2/2
    will_of_the_necropolis = 2/2

pvptalents: none

covenant: none

conduits: none

soulbinds: [plague_deviser_marileth]

sets: mirror_of_fractured_tomorrows = 1
    ominous_chromatic_essence = 1
    tier30 = 4

gear: arctic_wardens_gauntlets = 1
    chain_of_the_underking = 1
    frigid_conservators_wrap = 1
    lingering_phantoms_dreadhorns = 1
    lingering_phantoms_plackart = 1
    lingering_phantoms_schynbalds = 1
    lingering_phantoms_shoulderplates = 1
    mirror_of_fractured_tomorrows = 1
    ominous_chromatic_essence = 1
    primal_molten_greatbelt = 1
    primal_molten_sabatons = 1
    primal_molten_vambraces = 1
    renowned_guild_tabard = 1
    ringbound_hourglass = 1
    signet_of_titanic_insight = 1
    tailored_officers_shirt = 1
    thundercleaver_axe = 1
    tier30 = 4

legendaries: none

itemIDs: 133371, 134495, 190496, 190501, 190502, 192999, 193000, 202459, 202460, 202461, 202464, 203729, 207581, 210354, 210382, 69210, 89194

settings: aoe = 2
    combatRefresh = 0.25
    custom1Name = Custom 1
    custom2Name = Custom 2
    cycle = true
    cycleDebuff = festering_wound
    cycle_min = 6
    damage = true
    damageDots = false
    damageExpiration = 8
    damageOnScreen = true
    damagePets = false
    damageRange = 0
    enabled = true
    gcdSync = true
    maxTime = 20
    nameplateRange = 8
    nameplates = true
    noFeignedCooldown = false
    package = Unholy
    petbased = false
    potion = potion_of_spectral_strength
    regularRefresh = 0.5
    throttleRefresh = false
    throttleTime = false
    dps_shell = false

toggles: cooldowns = true 
    custom1 = false 
    custom2 = false 
    defensives = true 
    essences = true [overridden]
    interrupts = true 
    mode = automatic 
    potions = false [overridden]

keybinds: 
    abomination_limb    = V   [02]
    antimagic_shell     = 5   [01]
    antimagic_zone      = 3   [10]
    any_dnd             = 4   [01]
    apocalypse          = T   [02]
    army_of_the_dead    = S3  [01]
    asphyxiate          = SC  [01], SV  [08]
    blinding_sleet      = S4  [02]
    chains_of_ice       = SV  [01], SR  [07]
    clawing_shadows     = E   [02]
    control_undead      = C1  [05]
    dark_command        = 4   [10]
    dark_transformation = G   [02]
    death_coil          = R   [02]
    death_grip          = SR  [01]
    death_strike        = S2  [08]
    deaths_advance      = SX  [01]
    deaths_due          =
    defile              = 4   [01]
    empower_rune_weapon = S1  [01]
    epidemic            = C   [02]
    festering_strike    = Q   [02]
    healthstone         = SX  [07]
    icebound_fortitude  = 3   [01]
    lichborne           = 2   [01]
    mind_freeze         = F   [02]
    outbreak            = Z   [02]
    raise_dead          = S1  [09]
    scourge_strike      = E   [02]
    shadowmeld          = SF  [03]
    unholy_assault      = S2  [01]
    unholy_blight       = X   [02]
    wound_spender       = E   [02]

warnings: none
```

**Comment by Hekili** [comment]
Your snapshot is not complete.  You have to use CTRL+A to select the hidden text.  A full snapshot is hundreds/thousands of lines of text.  

A snapshot is required to diagnose this issue.

## Snapshot

Please supply a Snapshot of the addon's decision-making when you are seeing this issue in-game. This is not a screenshot. To generate a Snapshot, please complete the following steps.

 * Log into your WoW character.
 * Recreate the issue you are reporting (i.e., generate 5 Holy Power).
 * When you see the recommendation you disagree with, press ALT-SHIFT-P to Pause and Snapshot (or use ALT-SHIFT-[ to Snapshot without pausing).
   * You can change these keybindings in /hekili > Toggles if needed.
   * If you take the snapshot at a different time, it will not be useful and you'll be asked to submit a new one.
 * Type /hekili and press Enter.
 * Open the Snapshots section on the left side.
 * Select the snapshot that shows the recommendation that you disagree with.
   * Snapshots are labeled with your specialization, the display in question (usually, you'll choose Primary), and the recommendations the addon made with their timings.
 * Click in the Export Snapshot box, press CTRL+A to select all text including hidden text and press CTRL+C to copy.
 * Open [https://pastebin.com](https://github.com/Hekili/hekili/issues/Pastebin), use CTRL+V to paste the snapshot text, then click Create New Paste.
 * Provide the Pastebin ***link*** in a new comment.

This step is essential, as most issues are related to specific classes, specializations, gear, talent choices, or other game systems.  If you do not provide this information, I cannot triage your problem.

**Comment by burnshadowz** [comment]
> Your snapshot is not complete. You have to use CTRL+A to select the hidden text. A full snapshot is hundreds/thousands of lines of text.
> 
> A snapshot is required to diagnose this issue.
> 
> ## Snapshot
> Please supply a Snapshot of the addon's decision-making when you are seeing this issue in-game. This is not a screenshot. To generate a Snapshot, please complete the following steps.
> 
> * Log into your WoW character.
> * Recreate the issue you are reporting (i.e., generate 5 Holy Power).
> * When you see the recommendation you disagree with, press ALT-SHIFT-P to Pause and Snapshot (or use ALT-SHIFT-[ to Snapshot without pausing).
>   
>   * You can change these keybindings in /hekili > Toggles if needed.
>   * If you take the snapshot at a different time, it will not be useful and you'll be asked to submit a new one.
> * Type /hekili and press Enter.
> * Open the Snapshots section on the left side.
> * Select the snapshot that shows the recommendation that you disagree with.
>   
>   * Snapshots are labeled with your specialization, the display in question (usually, you'll choose Primary), and the recommendations the addon made with their timings.
> * Click in the Export Snapshot box, press CTRL+A to select all text including hidden text and press CTRL+C to copy.
> * Open [https://pastebin.com](https://github.com/Hekili/hekili/issues/Pastebin), use CTRL+V to paste the snapshot text, then click Create New Paste.
> * Provide the Pastebin _**link**_ in a new comment.
> 
> This step is essential, as most issues are related to specific classes, specializations, gear, talent choices, or other game systems. If you do not provide this information, I cannot triage your problem.

Hi Sorry about that this has been amended 
https://pastebin.com/38LtUXkZ 
i updated the link in the post

**Comment by Hekili** [comment]
This will be fixed in 1.0.1.
